intel-nuc.svg

```diff
@@ -1,32 +1 @@
-	</sodipodi:namedview>
-<g>
-	<path id="path6" fill="#127CC1" d="M444.329-86.082c-28.613-139.479-298.504-148.313-472.478-42.048v11.736
-		c173.745-89.683,420.28-89.11,442.716,39.371c7.558,42.564-16.252,86.831-58.943,112.303v33.332
-		C407.015,49.758,459.548-11.271,444.329-86.082 M127.971,158.123C7.907,169.237-117.193,151.746-134.707,57.557
-		c-8.698-46.378,12.475-95.604,40.405-126.146V-84.94c-50.361,44.328-77.717,100.398-61.922,166.598
-		c20.144,84.958,127.503,133.047,291.406,117.034c64.897-6.266,149.825-27.233,208.765-59.763V92.713
-		C290.384,124.792,201.793,151.294,127.971,158.123z"/>
-	<path id="path8" fill="#127CC1" d="M334.331-106.246h-31.473V34.559c0,16.538,7.897,30.918,31.473,33.199"/>
-	<path id="path10" fill="#127CC1" d="M-40.624-54.788h-31.476v91.966c0,16.542,7.898,30.922,31.476,33.202"/>
-	<rect id="rect12" x="-72.099" y="-101.967" fill="#127CC1" width="31.476" height="29.954"/>
-	<path id="path14" fill="#127CC1" d="M147.997,68.9c-25.52,0-36.279-17.803-36.279-35.374V-88.617h31.131v33.829h23.576v25.33
-		h-23.576v61.1c0,7.187,3.436,11.128,10.871,11.128h12.705V68.9H147.997"/>
-	<path id="path16" fill="#127CC1" d="M230.751-30.995c-10.643,0-18.883,5.533-22.317,13.007c-2.061,4.505-2.747,7.929-3.086,13.467
-		h48.184C252.845-18.045,246.773-30.995,230.751-30.995 M205.348,16.812c0,16.033,10.065,27.836,27.692,27.836
-		c13.851,0,20.719-3.878,28.73-11.8l19.23,18.539c-12.362,12.205-25.296,19.622-48.187,19.622
-		c-29.872,0-58.487-16.371-58.487-64.067c0-40.79,24.949-63.836,57.8-63.836c33.306,0,52.419,26.984,52.419,62.413v11.296h-79.198"
-		/>
-	<path id="path18" fill="#127CC1" d="M44.189-29.458c9.155,0,12.933,4.509,12.933,11.867V69.07H88.37v-86.774
-		c0-17.628-9.386-37.084-36.74-37.084h-64.438V69.068h31.131v-98.526"/>
-	<text transform="matrix(1.0217 0 0 1 352.1602 -85.6602)" fill="#127CC1" font-family="'ArialMT'" font-size="28.8313">®</text>
-</g>
-<text transform="matrix(1 0 0 1 -94.1338 408.9209)" fill="#127CC1" font-family="'Tahoma-Bold'" font-size="218.0193">NUC</text>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M16.25 6c.414 0 .75.336.75.75v9.5c0 .414-.336.75-.75.75h-9.5c-.414 0-.75-.336-.75-.75v-9.5c0-.414.336-.75.75-.75h9.5zm2.75 0c0-1.104-.896-2-2-2h-11c-1.104 0-2 .896-2 2v11c0 1.104.896 2 2 2h11c1.104 0 2-.896 2-2v-11zm-11 14v3h-1v-3h1zm4 0v3h-1v-3h1zm2 0v3h-1v-3h1zm-4 0v3h-1v-3h1zm6 0v3h-1v-3h1zm-8-20v3h-1v-3h1zm4 0v3h-1v-3h1zm2 0v3h-1v-3h1zm-4 0v3h-1v-3h1zm6 0v3h-1v-3h1zm4 15h3v1h-3v-1zm0-4h3v1h-3v-1zm0-2h3v1h-3v-1zm0 4h3v1h-3v-1zm0-6h3v1h-3v-1zm-20 8h3v1h-3v-1zm0-4h3v1h-3v-1zm0-2h3v1h-3v-1zm0 4h3v1h-3v-1zm0-6h3v1h-3v-1z"/></svg>
```

Comment:
@telphan , is this the icon that @konmouz provided?

Reply:
@floion, yes it is